web/client/selectors/featuregrid.js

```diff
@@ -184,3 +184,4 @@ export const isEditingAllowedSelector = state => {
     return (editingAllowedRoles.indexOf(role) !== -1 || canEdit) && !isCesium(state);
 };
 export const paginationSelector = state => get(state, "featuregrid.pagination");
+export const useLayerFilterSelector = state => get(state, "featuregrid.useLayerFilter") ?? true;
```

Comment:
what about this? if the values is false it will be taken, so no need to use ?? here
right?
```suggestion
export const useLayerFilterSelector = state => get(state, "featuregrid.useLayerFilter", true);
```